spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -20,6 +20,12 @@ class Klass
         create(:submitted_feedback, user: user, review: invitations.first.review)
         klass.open_requests(user).should == 4
       end
+
+      it 'should include invitations for reviews that have other feedback' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.